### DISCUSSION TITLE
Is it a bad idea to increase the frequency of updates?

### DISCUSSION THREAD

**Discussion by voidrender** [upptime/upptime#545] [Q&A] [answered]
I realize that some of the updates commit to git so this will have impacts on the repo size, but is it a bad idea to have, say, the response time Action running every 5 minutes instead of daily? Any long-term considerations?

**Comment by AnandChowdhary** [comment, marked as answer]
Apart from the thousands of commits, I can't think of any other reason. Some repositories are doing this already, but I think it's not great because the Graphs CI generates PNGs which then are updated and committed every few minutes, which increases the repo size by a lot. You might hit the GitHub 100 GB limit after a while then.
